_posts/2017-04-17-wego.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Wego
+sourceURL: http://saudigazette.com.sa/business/3-seconds-better-12/
+image: wego.png
```

Comment:
This must have changed after I pulled the repo because I used twitter as a model and my local copy of twitter doesn't have a path from root. Regardless, it is updated for Wego now.